pkg/agent/agent.go

```diff
@@ -428,12 +428,16 @@ func (i *Initializer) setupDefaultTunnelInterface(tunnelPortName string) error {
 
 	// Create default tunnel port.
 	if i.networkConfig.TrafficEncapMode.SupportsEncap() {
-		tunnelPortUUID, err := i.ovsBridgeClient.CreateTunnelPort(tunnelPortName, i.networkConfig.TunnelType, config.DefaultTunOFPort)
+		var localIP net.IP
+		if i.useFixedTunnelLocalIP() {
```

Comment:
This func sounds strange. Do you feel better to move line 431 to line 435 to a separate func like createDefaultTunnelPort() which returns localIP and tunnelPortUUID, and has different implementations in agent_linux.go and _windows.go?

Reply:
OK. I add createDefaultTunnelPort() for different platform.